clerk/templates.go

```diff
@@ -69,13 +70,15 @@ type UpsertTemplateRequest struct {
 	Markup           string  `json:"markup,omitempty"`
 	Body             string  `json:"body"`
 	FromEmailName    *string `json:"from_email_name"`
+	ReplyToEmailName *string `json:"reply_to_email_name"`
```

Comment:
:question: Should we also `omitempty`?

Reply:
@gkats looks like I have to resolve this conversation before merging, so I'll go ahead and do so! Already updated to use `omitempty` here